webhook/resources/volume/mutator.go

```diff
@@ -163,20 +172,22 @@ func (v *volumeMutator) Create(request *admission.Request, newObj runtime.Object
 			return nil, werror.NewInvalidError(fmt.Sprintf("get invalid size for volume %v: %v", backup.Status.VolumeSize, err), "")
 		}
 
-		// Add backup volume name label to the restore/DR volume
-		labels := volume.Labels
-		if labels == nil {
-			labels = map[string]string{}
-		}
```

Comment:
Should we retain this for existing volumes?

Reply:
Yes, I think so. Therefore, I moved this variable initialization to line 120, even though the volume was restored from a backup.